Presentation.Web/Swagger/SwashbuckleExtensions.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Swashbuckle.Swagger;
+using System.Collections.Generic;
+
```

Comment:
We already have an integration test that covers the V1 and V2 json files.
Extend that suite with two tests that verify that a v2 dto is not present in the v1 json but IS in the v1 json and vice versa.